test/test_integration_pr2.py

```diff
@@ -1603,6 +1603,77 @@ def test_open_close_dishwasher(self, kitchen_setup: PR2TestWrapper):
+                                     door_name)  # root link of the objects to avoid collision
+        kitchen_setup.set_env_state({'sink_area_dish_washer_door_joint': np.pi/8})
+        tip_grasp_axis = Vector3Stamped()
+        tip_grasp_axis.header.frame_id = hand
+        tip_grasp_axis.vector.y = 1
+
+        object_rotation_axis = Vector3Stamped()
+        object_rotation_axis.header.frame_id = door_name
+        object_rotation_axis.vector.z = 1
+
+        kitchen_setup.set_align_to_push_door_goal(root_link=kitchen_setup.default_root,
+                                                  tip_link=hand,
+                                                  door_object=door_name,
+                                                  door_height=0.6,
+                                                  object_joint_name=door_joint,
+                                                  door_length=0.49,
+                                                  tip_gripper_axis=tip_grasp_axis,
+                                                  object_rotation_axis=object_rotation_axis)
+
+        kitchen_setup.plan_and_execute()
+        object_normal = Vector3Stamped()
+        object_normal.header.frame_id = door_name
+        object_normal.vector.x = 1
+
+        # # # close the gripper
+        kitchen_setup.set_joint_goal(goal_state={'r_gripper_l_finger_joint': 0.0})
+
+        root_V_object_rotation_axis = Vector3Stamped()
+        root_V_object_rotation_axis.header.frame_id = "map"
+        root_V_object_rotation_axis.vector.y = -1
+
+        kitchen_setup.set_pre_push_door_goal(root_link=kitchen_setup.default_root,
+                                             tip_link=hand,
+                                             door_object=door_name,
+                                             door_height=0.6,
+                                             door_length=0.49,
+                                             door_depth=0.02,
+                                             tip_gripper_axis=tip_grasp_axis,
+                                             root_V_object_rotation_axis=root_V_object_rotation_axis,
+                                             object_joint_name=door_joint)
+
+        kitchen_setup.allow_collision(group1=door_obj, group2=kitchen_setup.r_gripper_group)
+        kitchen_setup.plan_and_execute()
+
+        right_forearm = 'r_forearm'
+        kitchen_setup.register_group(right_forearm,
+                                     root_link_group_name=kitchen_setup.robot_name,
+                                     root_link_name='r_forearm_link')
+        kitchen_setup.set_open_container_goal(tip_link=hand,
+                                              environment_link=handle_name,
+                                              goal_joint_state=1.3217)
+
+        kitchen_setup.allow_collision(group1=door_obj, group2=right_forearm)
+        kitchen_setup.plan_and_execute()
```

Comment:
this test only checks if the goal is parsed correctly, not if it is executed correctly. can you check automatically, if it's doing the right thing?